pkgs/development/python-modules/nltk.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchurl, buildPythonPackage, isPy33, lib, six, pythonAtLeast, pythonOlder }:
+
+buildPythonPackage rec {
+  name = "nltk-${version}";
+  version = "3.2.2";
+
+  src = fetchurl {
+    url = "mirror://pypi/n/nltk/nltk-${version}.tar.gz";
+    sha256 = "13m8i393h5mhpyvh5rghxxpax3bscv8li3ynwfdiq0kh8wsdndqv";
+  };
+
+  buildInputs = [ six ];
```

Comment:
Is six really only required at build time? If not it should be in `propagatedBuildInputs` instead.